<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalystWindow</class>
 <widget class="QMainWindow" name="AnalystWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SWOT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Проект</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Выберите проект</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="questionThemeComboBox"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="themeTestButton">
          <property name="text">
           <string>Просмотреть отчет</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QLabel" name="errorTestLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="genetalTestButton">
          <property name="text">
           <string>Создать новый проект</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Название проекта</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Weaknesses</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="7" column="2" colspan="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Отправить</string>
          </property>
         </widget>
        </item>
        <item row="6" column="8">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="6" column="7">
         <widget class="QSpinBox" name="spinBox_10"/>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Action</string>
          </property>
         </widget>
        </item>
        <item row="6" column="5" colspan="2">
         <widget class="QSpinBox" name="spinBox_9"/>
        </item>
        <item row="3" column="5" colspan="2">
         <widget class="QSpinBox" name="spinBox_3"/>
        </item>
        <item row="5" column="7">
         <widget class="QSpinBox" name="spinBox_8"/>
        </item>
        <item row="3" column="8">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="5" column="8">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="3">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="2" column="5" colspan="2">
         <widget class="QSpinBox" name="spinBox"/>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
        <item row="4" column="7">
         <widget class="QSpinBox" name="spinBox_6"/>
        </item>
        <item row="2" column="7">
         <widget class="QSpinBox" name="spinBox_2"/>
        </item>
        <item row="3" column="7">
         <widget class="QSpinBox" name="spinBox_4"/>
        </item>
        <item row="4" column="8">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="3">
         <widget class="QLineEdit" name="lineEdit_7"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_8"/>
        </item>
        <item row="4" column="5" colspan="2">
         <widget class="QSpinBox" name="spinBox_5"/>
        </item>
        <item row="5" column="2" colspan="3">
         <widget class="QLineEdit" name="lineEdit_9"/>
        </item>
        <item row="2" column="8">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="5" colspan="2">
         <widget class="QSpinBox" name="spinBox_7"/>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_6"/>
        </item>
        <item row="3" column="2" colspan="3">
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
        <item row="6" column="2" colspan="3">
         <widget class="QLineEdit" name="lineEdit_11"/>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_10"/>
        </item>
        <item row="0" column="8">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Power</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Probability</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Importance</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Page</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Личный кабинет</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>594</width>
             <height>374</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Фото</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QGraphicsView" name="photoView"/>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Загрузить фото</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Пароль*</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="passwordLineEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Имя*</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="nameLine"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Фамилия*</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="surnameLine"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Отчество</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="faternityLine"/>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Дата рождения*</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Группа*</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QComboBox" name="group">
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Секретный вопрос*</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QComboBox" name="question"/>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Ответ*</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QLineEdit" name="answerLine"/>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Эл. почта*</string>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QLineEdit" name="emailLine"/>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Телефон</string>
              </property>
             </widget>
            </item>
            <item row="11" column="1">
             <widget class="QLineEdit" name="phoneLine"/>
            </item>
            <item row="12" column="1">
             <widget class="QPushButton" name="acceptChangesButton">
              <property name="text">
               <string>Подтвердить изменения</string>
              </property>
             </widget>
            </item>
            <item row="15" column="0" colspan="2">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>После внесения изменений, потребуется перезайти</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="16" column="0" colspan="2">
             <widget class="QLabel" name="errorChangesLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="dateOfBirthLine"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Статистика</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>607</width>
             <height>340</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="statisticLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="viewGraphicsButton">
              <property name="text">
               <string>Отобразить графики</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Вопросная база</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Группа вопросов:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="groupsOfQuestionsBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Список вопросов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="questionsComboBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="editQuestionButton">
          <property name="text">
           <string>Редактировать вопрос</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="labelAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <item row="1" column="0">
           <widget class="QPushButton" name="hideQuestionButton">
            <property name="text">
             <string>Скрыть вопрос</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="hideGroupOfQuestionsButton">
            <property name="text">
             <string>Скрыть группу Вопросов</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="deleteGroupOfQuestionsButton">
            <property name="text">
             <string>Удалить группу вопросов</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="deleteQuestionButton">
            <property name="text">
             <string>Удалить вопрос</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="AddQuestionButton">
          <property name="text">
           <string>Добавить вопрос</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
